apps/dashboard/app/helpers/projects_helper.rb

```diff
@@ -14,4 +14,21 @@ def render_readme(readme_location)
       simple_format(file_content)
     end
   end
-end
\ No newline at end of file
+
+  def job_details_buttons(status, job)
+    locals = { project_id: @project&.id, id: job&.id, cluster: job&.cluster }
```

Comment:
Seems like you could jsut pass the project into t his method to avoid the safe traversal (`&`).  That's throwing me off - like we know `@project` can be nil here (but really shouldn't be?) which is a very odd state to be in. 

Or maybe we can add a `before_action` to set `@project`, but I would prefer helpers like this be purely functional just to be safe/make it easy for us to reason about it without having to know what state may or may not be available.